src/server/storage/storagehelpers.js

```diff
@@ -89,6 +141,17 @@ function loadPath(dbProject, rootHash, loadedObjects, path, excludeParents) {
     var deferred = Q.defer(),
         pathArray = path.split('/');
 
+    function processLoadResult(hash, result) {
+        var subHash;
+        if (result.hasOwnProperty('multipleObjects') && result.multipleObjects === true) {
+            for (subHash in result.objects) {
+                loadedObjects[subHash] = result.objects(subHash);
```

Comment:
Check the coverage of these:
![image](https://cloud.githubusercontent.com/assets/6518904/23708747/a31b2c10-03dc-11e7-802d-fe5631379bf3.png)